src/content/documentation/develop/best-practices-for-collecting-user-data-consents.md

```diff
@@ -152,7 +172,7 @@ You can prompt the user to uninstall the extension with [`management.uninstallSe
 
 ## Your consent dialogs
 
-Once you understand your privacy consent flow, you can add a suitable consent dialog. The following are suggestions on how to present a data collection consent dialog. Where the mockups include the linked text “our Privacy Policy”, this should go to your extension’s privacy policy page on AMO. If you choose to follow these mockups, remember to modify them appropriately for your extension.
+Once you understand your privacy consent flow, you can add a suitable consent dialog. The following are suggestions on how to present a data collection consent dialog. We recommend that all of the data collection controls and actions (that is, toggles and confirmation buttons) be present within the same view. Where the mockups include the linked text “our Privacy Policy”, this should go to your extension’s privacy policy page on AMO. If you choose to follow these mockups, remember to modify them appropriately for your extension.
```

Comment:
Do the mockups and flow diagrams respect this recommendation?

Reply:
I'd say they're slightly off from our recommendation here, especially the last image which has a "revisit data collection consent" button, implying multiple views. 

Do you know how the original images were created, or if we're stuck to a particular style? We should update the images to be in line with our recommendations, even if they're just recommendations (I hadn't originally planned on changing images, but it makes sense to do it together).

Reply:
Unfortunately I don't know. I agree, ideally we should update the diagrams when they are not accurate.

Reply:
@rebloor can you provide us with the original source files for the images, so we can update them? Can you add them to https://drive.google.com/drive/folders/1Y_Ap1kbMaQCsMI2E_pbJ-b46zvu0Re3o?

Reply:
> @rebloor can you provide us with the original source files for the images, so we can update them?
Sorry, I missed this request. Unfortunately, I'm not aware of the whereabouts of these files and couldn't find a copy. However, I see that the mock ups have now being re-created.

Reply:
@wagnerand @abhn I've added a copy of the flow diagram to the shared GDrive - it should be the latest.